pyproject.toml

```diff
@@ -39,14 +40,14 @@ classifiers = [
 ]
 requires-python = '>=3.7'
 dependencies = [
-    'pydantic>=2.0.3',
+    'pydantic>=2.4.2',
```

Comment:
Is this necessary?

Reply:
I guess to match and support the latest changes in pydantic so far 🤔